packages/live-preview-sdk/src/__tests__/csm.spec.ts

```diff
@@ -77,10 +77,26 @@ describe('Content Source Maps', () => {
         title: {
           origin: 'contentful.com',
           href: 'https://app.contentful.com/spaces/foo/environments/master/entries/a1b2c3/?focusedField=title&focusedLocale=en-US',
+          cf: {
```

Comment:
Sorry but what does `cf` mean here? is there a more meaningful property name you could give it?

Reply:
contentful?

Reply:
Funnily, I feel like in this repo we seem to overly express attribute names e.g. `InspectorModeDataAttributes`. in here I went to the other extreme mostly to save a few bytes but I guess that should have been applied consistently.